compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -18,7 +18,7 @@
 
 #include <gtest/gtest.h>
 
-TEST(ResolveCustomOpAddPassTest, name)
+TEST(ResolveAddV2Test, name)
```

Comment:
Related discussion
- The discussion of the test name came from [here](https://github.com/Samsung/ONE/pull/12509#discussion_r1464360947).

Reply:
`ResolveAddV2Test` doesn't seem to be the correct name.
Have you tried introduce a dummy `ResolveCustomOpAddPassTest` class and use `TEST_F` ?

Reply:
@seanshpark 
> Have you tried introduce a dummy ResolveCustomOpAddPassTest class and use TEST_F ?

Maybe I don't understand well 😅, 
are you talking about introducing the following changes?

```
#include "luci/Pass/ResolveCustomOpAddPass.h"

#include <gtest/gtest.h>
#include <luci/test/TestIOGraph.h>
#include <luci/IR/CircleNodes.h>

using namespace luci::test;

namespace
{

class ResolveCustomOpAddPassTest : public ::testing::Test
{
public:
  luci::ResolveCustomOpAddPass _pass;
};

} // namespace

TEST_F(ResolveCustomOpAddPassTest, name)
{
  auto const name = _pass.name();
  ASSERT_NE(nullptr, name);
}
```

Reply:
> are you talking about introducing the following changes?

Looks like it :)

Reply:
@seanshpark 

Oh, I just tried building with that change, and it seems to be working well.
I will revise the code and upload it again. 
Thank you a lot :)

Reply:
I uploaded it. :)